google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -828,7 +820,7 @@ public static class Builder
     private OpenTelemetry openTelemetry;
     private boolean enableApiTracing = SpannerOptions.environment.isEnableApiTracing();
     private boolean enableExtendedTracing = SpannerOptions.environment.isEnableExtendedTracing();
-    private boolean enableBuiltInMetrics = SpannerOptions.environment.isEnableBuiltInMetrics();
+    private boolean enableBuiltInMetrics = true;
```

Comment:
super nit: could we move this line down or up a bit, so it is not in the middle of the properties that are read from the environment? That makes the grouping of the (long) list of properties a bit easier to read.